tests/epw_test.py

```diff
@@ -47,9 +47,23 @@ def test_import_tokyo_epw(self):
         assert epw.is_data_loaded is True
         assert len(dbt) == 8760
 
-    def test_epw_location(self):
-        """Test epw location."""
-        pass
+    def test_save_epw(self):
+        """Test save epw_rel."""
+        path = './tests/epw/tokyo.epw'
+        epw = EPW(path)
+
+        modified_path = './tests/epw/tokyo_modified.epw'
+        epw.save(modified_path)
+        assert os.path.isfile(modified_path)
+
+    def test_save_wea(self):
+        """Test save wea_rel."""
+        path = './tests/epw/chicago.epw'
+        epw = EPW(path)
+
+        wea_path = './tests/wea/chicago_epw.wea'
+        epw.to_wea(wea_path)
+        assert os.path.isfile(wea_path)
```

Comment:
I would add checks to make sure the order of the data matched what is expected. This business of shifting data an hour back and forth can end up making issues in cases like this and adding a test will ensure that it works fine.

Reply:
Good idea.  Thankfully, all of the radiation and illuminance data never gets shifted (so the Wea should be fine in this case).  Still, I'm adding it in now.

Reply:
Done